tools/test.py

```diff
@@ -316,6 +316,12 @@ def HasRun(self, output):
       sys.stdout.flush()
 
 class ActionsAnnotationProgressIndicator(DotsProgressIndicator):
+  def AboutToRun(self, case):
+    if not hasattr(case, 'additional_flags'):
+      case.additional_flags = []
+    case.additional_flags.append('--test-reporter=./tools/github_reporter/index.js')
+    case.additional_flags.append('--test-reporter-destination=stdout')
```

Comment:
If I understand this correctly, this is adding two additional flags to _all_ the tests that run. This feels worrisome, because as a test author I’d expect the flags I define at the top of the test file to be the only flags that are used; I can imagine issues with `process.argv` and so on. Could these be defined via environment variables instead?

Reply:
this is only the case in github actions, within jenkins or locally the flags wont be added.
the only way to add this via an environment variable is with `NODE_OPTIONS`